components/molecule/select/test/index.test.js

```diff
@@ -100,11 +100,11 @@ describe(json.name, () => {
       const {getByRole} = setup(props)
       const textBox = getByRole('textbox')
 
-      userEvents.click(textBox)
-      userEvents.tab()
+      fireEvent.focusIn(textBox)
+      fireEvent.focusOut(textBox)
```

Comment:
https://testing-library.com/docs/guide-events

have a look at this... perhaps that's why you're feeling it's a bit flaky on your local environment? 🤔  

perhaps using imperative focus / blur is more robust:
https://testing-library.com/docs/guide-events#alternatives

Reply:
Interesting! Thanks for sharing! 💡 

I have given it a read and updated this with the imperative `focus()`

Seems like for some reason, calling imperatively `blur()` does not seem to work as you'd expect 🤔 
![Captura de Pantalla 2022-10-14 a las 10 11 29](https://user-images.githubusercontent.com/22222078/195797310-07c7df95-6cbb-4b41-9881-b1321a9d54d8.png)
![Captura de Pantalla 2022-10-14 a las 10 11 08](https://user-images.githubusercontent.com/22222078/195797323-cd6c0b26-49a3-4eaf-bb3b-17c6851ca1ef.png)

Reply:
oh that's sad... 🤔 

as an interim solution... how do you feel about doing focus first on `textBox`, as you're already doing, and then on some other element so that it _hopefully_ fires the blur event on the textbox? I know it's a bit hacky but perhaps more robust than the original `fireEvent` focus and blur.

Reply:
Better use click and tab, Its what user is going to tackle, dont care if it blurs or focus on his/her POV

Reply:
I agree, `click` and `tab` should be what we aim for, which was my original intention

```jsx
userEvents.click(textBox)
userEvents.tab()
```

After trying quite different things, like rendering another component and clicking that component 

```jsx

      const component = props => (
        <>
          <div>
            <button>Hello!</button>
          </div>
          <Component {...props} />
        </>
      )

      const setup = setupEnvironment(component)
```

I've realized that this behavior happens whether the browser that runs the tests is opened/closed. 🤯 

![ezgif com-gif-maker (1)](https://user-images.githubusercontent.com/22222078/195822749-07ab4154-0d41-4599-8f7c-6aa9cccf6421.gif)

only with `fireEvent.blur()` it fires the event from the node, regardless of wether the Karma browser is opened or not...